Source/WebKit/UIProcess/Extensions/WebExtension.cpp

```diff
@@ -1406,7 +1407,17 @@ void WebExtension::populateContentScriptPropertiesIfNeeded()
         }
 
         // Optional. Whether the script should inject into an about:blank frame where the parent or opener frame matches one of the patterns declared in matches. Defaults to false.
-        auto matchesAboutBlank = injectedContentObject->getBoolean(contentScriptsMatchesAboutBlankManifestKey).value_or(false);
+        bool matchAboutBlank = injectedContentObject->getBoolean(contentScriptsMatchAboutBlankManifestKey).value_or(false);
+
+        // Optional. Whether the script should inject in frames that were created by a matching origin, but whose URL or origin may not directly match the pattern.
+        // These include frames with different schemes, such as about:, data:, and blob:. Defaults to false.
+        bool matchOriginAsFallback = injectedContentObject->getBoolean(contentScriptsMatchOriginAsFallbackManifestKey).value_or(false);
+
+        auto matchParentFrame = WebCore::UserContentMatchParentFrame::Never;
```

Comment:
Super-nit: I was curious if matchOriginAsFallback -> matchAboutBlank was the correct fallback, and this was defined in: https://developer.chrome.com/docs/extensions/develop/concepts/content-scripts (`When both "match_origin_as_fallback" and "match_about_blank" are specified, "match_origin_as_fallback" takes priority.`)

Could be worth mentioning that in a comment.